pkg/ingester/ingester.go

```diff
@@ -3748,8 +3766,22 @@ func (i *Ingester) PushToStorage(ctx context.Context, req *mimirpb.WriteRequest)
 	return nil
 }
 
+func (i *Ingester) isCircuitBreakerActive() bool {
+	if i.circuitBreaker == nil {
```

Comment:
We could move `nil`-check into `circuitBreaker.isActive()` method and drop this method.